common/src/main/resources/common/add-okapi-permissions.feature

```diff
@@ -0,0 +1,74 @@
+Feature: prepare data for api test
+
+  Background:
+    * url baseUrl
+    * call login admin
+    * def adminToken = okapitoken
+    * configure headers = { 'Content-Type': 'application/json', 'Accept': 'application/json', 'x-okapi-token' : #(adminToken)}
+    * table okapiPermissionsTable
+      | name                              |
+      |'okapi.proxy.modules.post'|
+      |'okapi.proxy.modules.put'|
+      |'okapi.proxy.modules.delete'|
+      |'okapi.proxy.pull.modules.post'|
+      |'okapi.proxy.tenants.put'|
+      |'okapi.proxy.tenants.delete'|
+      |'okapi.proxy.tenants.modules.post'|
+      |'okapi.proxy.tenants.modules.enabled.post'|
+      |'okapi.proxy.tenants.upgrade.post'|
+      |'okapi.proxy.tenants.install.post'|
+      |'okapi.proxy.tenants.modules.enabled.delete'|
+      |'okapi.env.delete'|
+      |'okapi.env.list'|
+      |'okapi.modules'|
+      |'okapi.env'|
+      |'okapi.deployment.post'|
+      |'okapi.deployment.get'|
+      |'okapi.deployment.delete'|
+      |'okapi.discovery.post'|
+      |'okapi.discovery.get'|
+      |'okapi.discovery.put'|
+      |'okapi.discovery.delete'|
+      |'okapi.discovery.nodes.put'|
+      |'okapi.discovery.health.get'|
+      |'okapi.discovery.nodes.get'|
+      |'okapi.env.post'|
+      |'okapi.env.get'|
+      |'okapi.all'|
```

Comment:
Does okapi.all includes all other permissions mentioned in this list?

Reply:
removed redudant permissions, thanks